### DISCUSSION TITLE
公有云同一地区下的RDS无法同时监控

### DISCUSSION THREAD

**Discussion by weekday** [starsliao/ConsulManager#89] [Q&A] [answered]
阿里云深圳区，有两台mysql RDS 和一台 PGSQL RDS，consul能获取到三台RDS信息，接入监控后，grafana中只能显示一台，另两台无监控数据且一直告警 multi_mysqld_exporter异常

**Comment by starsliao** [comment]
应该是不会发现pg的，另外multi_mysqld_exporter异常 你需要查看multi_mysqld_exporter的日志

**Comment by weekday** [comment]
consul上面确实已经发现了PGSQL，如图：
![image](https://github.com/starsliao/TenSunS/assets/15261940/9e814fc3-95b8-415e-bc01-9cccc8bd2d6b)

multi_mysqld_exporter日志不断报这个错误，如图：![image](https://github.com/starsliao/TenSunS/assets/15261940/e6769a77-554b-4231-9a1d-af2d506e766e)

**Reply by starsliao** [reply, marked as answer]
你好,已经修复了这个bug,阿里云会只采集mysql的实例,请更新下镜像
`docker-compose pull && docker-compose pull  up -d`

**Comment by weekday** [comment]
现在确实已经剔除了PGSQL的采集，但是mysqld_exporter对阿里云深圳区的另一个mysql RDS的监控任然是报错。
![image](https://github.com/starsliao/TenSunS/assets/15261940/36fb7826-a158-48ca-8c5e-0604574d96ca)

**Reply by starsliao** [reply]
这个是监控的报错了, prometheus去请求mysql-exporter,然后mysql-exporter再去请求这个mysql实例的报错, 看看是不是网络不通,或者权限问题.

**Reply by weekday** [reply]
多谢，已经解决
